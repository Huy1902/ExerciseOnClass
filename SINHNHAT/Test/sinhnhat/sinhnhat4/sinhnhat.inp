12 6
150 52 70 157 145 194 164 158 76 102 168 180 