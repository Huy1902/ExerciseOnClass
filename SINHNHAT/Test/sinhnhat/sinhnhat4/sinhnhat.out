1021
4 6 7 8 11 12 